tests/e2e/iks-cluster

```diff
@@ -68,7 +68,7 @@ PLATFORM_kubernetes="k8s"
 PLATFORM_openshift="ocp"
 
 PRIMARY_ZONE_ID="01"	# ZONE_vpc_gen2_01 / ZONE_classic_01
-SECOND_ZONE_ID="03"	# ZONE_vpc_gen2_03 / ZONE_classic_03
+SECOND_ZONE_ID="02"	# ZONE_vpc_gen2_03 / ZONE_classic_03
```

Comment:
@GunaKKIBM change comment accordingly

Reply:
@GunaKKIBM  Could you explain reason for this change.